db/migrate/20170319193517_add_badge_id_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddBadgeIdToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :badge_id, :integer
+    add_column :users, :badge_id, :integer, default: 0
+    User.where(donor: true).update_all(badge_id: 1)
```

Comment:
Oh, I just assumed we would make the badges. Thanks for the tip, this will come in handy later on, I'm sure.